avocado/plugins/runner_nrunner.py

```diff
@@ -74,7 +74,7 @@ def initialize(self):
                     'is in another host, or different port')
         settings.register_option(section=section,
                                  key='status_server_uri',
-                                 default='127.0.0.1:8888',
+                                 default=None,
```

Comment:
I can't see a problem with having this as default value. Can you explain better, what problem are you solving here, please?